package.json

```diff
@@ -13,8 +13,8 @@
     "test": "test"
   },
   "dependencies": {
-    "object-assign": "^4.1.0",
```

Comment:
why did you change the `object-assign` dependency?

Reply:
When I tested the changes, npm was unable to locate the existing dependencies so it required a fresh install of the dependencies listed in the package.json and must have updated that particular dependency. Would you like me to revert to `4.1.0`?

Reply:
yes, ideally this PR should only contain the changes related to `stringz`

Reply:
Sounds good. `object-assign` was reverted to `^4.1.0` using `yarn`